src/CabanaPD_Solver.hpp

```diff
@@ -192,15 +192,13 @@ class SolverElastic
     void init( const bool initial_output = true )
     {
         // Compute/communicate LPS weighted volume (does nothing for PMB).
+        // Only computed once.
         force->computeWeightedVolume( *particles, *neighbors,
                                       neigh_iter_tag{} );
         comm->gatherWeightedVolume();
-        // Compute/communicate LPS dilatation (does nothing for PMB).
-        force->computeDilatation( *particles, *neighbors, neigh_iter_tag{} );
-        comm->gatherDilatation();
-
         // Compute initial forces.
-        computeForce( *force, *particles, *neighbors, neigh_iter_tag{} );
```

Comment:
What is the main difference between "computeForce" and "updateForce"?

Reply:
Check the new `updateForce` - it calls compute and all other force-related functions